doc/get-involved.md

```diff
@@ -3,14 +3,13 @@ title = "Get involved"
 docpage = true
 [menu.docs]
   parent = "get-involved"
-
 +++
 
-We're really excited to welcome Contributors to [Conduit](https://github.com/linkerd/linkerd2)!
+We're really excited to welcome Contributors to [Linkerd](https://github.com/linkerd/linkerd2)!
 
 There are several ways to get involved:
 
-- Conduit on [GitHub](https://github.com/linkerd/linkerd2)
+- Linkerd on [GitHub](https://github.com/linkerd/linkerd2)
 - Join us on the #conduit channel in [Linkerd slack](https://slack.linkerd.io/)
```

Comment:
Related to the comment above, we should either rename the Slack channel to #linkerd2 and rename the channel here, or we should revert the channel name change above.